layout/templates/textSection.mustache

```diff
@@ -1,4 +0,0 @@
-<div class="viz-text">
```

Comment:
confused..is this just extra garbage you are cleaning up or did it get mixed up in this PR accidentally?

Reply:
That is a cleanup, we aren't using a template for the text, so not needed.

Reply:
I shoulda explained that.

Reply:
No worries - just wanted to make sure it wasn't swept up in a merge conflict resolution or something!